pkg/proxy/iptables/proxier.go

```diff
@@ -196,6 +197,10 @@ type Proxier struct {
 	// localhostNodePorts indicates whether we allow NodePort services to be accessed
 	// via localhost.
 	localhostNodePorts bool
+
+	// tcpLiberal indicates whether the system sets the kernel nf_conntrack_tcp_be_liberal
+	tcpLiberal bool
```

Comment:
maybe `conntrackTCPLiberal` or something like that... it's only about conntrack behavior, not TCP behavior in general